pkg/loop/config.go

```diff
@@ -11,13 +11,20 @@ import (
 )
 
 const (
-	envDatabaseURL            = "CL_DATABASE_URL"
-	envPromPort               = "CL_PROMETHEUS_PORT"
+	envDatabaseURL = "CL_DATABASE_URL"
+	envPromPort    = "CL_PROMETHEUS_PORT"
+
 	envTracingEnabled         = "CL_TRACING_ENABLED"
 	envTracingCollectorTarget = "CL_TRACING_COLLECTOR_TARGET"
 	envTracingSamplingRatio   = "CL_TRACING_SAMPLING_RATIO"
 	envTracingAttribute       = "CL_TRACING_ATTRIBUTE_"
 	envTracingTLSCertPath     = "CL_TRACING_TLS_CERT_PATH"
+
+	envTelemetryEndpoint         = "CL_TELEMETRY_ENDPOINT"
+	envTelemetryInsecureConn     = "CL_TELEMETRY_INSECURE_CONNECTION"
+	envTelemetryCACertFile       = "CL_TELEMETRY_CA_CERT_FILE"
+	envTelemetryAttribute        = "CL_TELEMETRY_ATTRIBUTE_"
+	envTelemetryTraceSampleRatio = "CL_TELEMETRY_TRACE_SAMPLE_RATIO"
```

Comment:
#nit to have the same spelling as `CL_TRACING_SAMPLING_RATIO`

```suggestion
	envTelemetryTraceSamplingRatio = "CL_TELEMETRY_TRACE_SAMPLING_RATIO"
```

Reply:
I think we need to prioritize aligning the field end to end, rather than with other sections. Do we want to rename this everywhere?

Reply:
Yes, if you rename env vars here then renaming beholder config fields as will be necessary as well